src/components/common.js

```diff
@@ -228,3 +228,31 @@ export const LargeFadeBox = ({ children }) => {
     }, [children])
   ])
 }
+
+const viewToggleStyles = {
+  toolbarContainer: {
+    flex: 'none', display: 'flex', alignItems: 'flex-end',
+    color: colors.blue[0], padding: '0 2.25rem'
```

Comment:
nit: `color` and `alignItems` are unnecessary here.

Also, generally speaking I try to avoid having external padding/margins on shared components, because it forces the consumer to deal with that, and can cause cascading alignment headaches down the line.

Reply:
I removed `color` and `alignItems`. I'll keep that in mind with padding and margin, I can see how it could cause headaches downstream.